README.md

```diff
@@ -31,6 +31,7 @@ bcache | Exposes bcache statistics from `/sys/fs/bcache/`. | Linux
 bonding | Exposes the number of configured and active slaves of Linux bonding interfaces. | Linux
 boottime | Exposes system boot time derived from the `kern.boottime` sysctl. | Darwin, Dragonfly, FreeBSD, NetBSD, OpenBSD, Solaris
 conntrack | Shows conntrack statistics (does nothing if no `/proc/sys/net/netfilter/` present). | Linux
+cooling\_device | Exposes cooling device statistics from `/sys/class/thermal`. | Linux
```

Comment:
Is it worth having this be a separate collector from `thermal_zone`? Maybe we should just have `--collector.thermal` that handles all metrics from `/sys/class/thermal`?

Reply:
I think that makes sense, but I'm not sure how best to achieve this. For now I'll move most of the code from this PR into the thermal collector file - there should be a one-to-one mapping between the two anyway. Let me know if this approach seems off, otherwise I should have this updated soon.

Reply:
> LGTM!
> I assume the 'throttle state' is just an arbitrary unit-less number? If so, all good. If not, let's make this more self-explanatory.

Yup, pretty much. The number is between 0 (or -1 for Intel powerclamp devices) and max_state; but they are unit-less and totally arbitrary.